bsp4j/src/main/xtend-gen/ch/epfl/scala/bsp4j/ScalaTestClassesItem.java

```diff
@@ -11,21 +11,14 @@ public class ScalaTestClassesItem {
   @NonNull
   private BuildTargetIdentifier target;
 
+  private String framework;
+
   @NonNull
   private List<String> classes;
 
-  private String framework;
-
   public ScalaTestClassesItem(@NonNull final BuildTargetIdentifier target, @NonNull final List<String> classes) {
     this.target = target;
     this.classes = classes;
-    this.framework = null;
-  }
-
-  public ScalaTestClassesItem(@NonNull final BuildTargetIdentifier target, @NonNull final List<String> classes, final String framework) {
```

Comment:
Looks like this just removes a convenience constructor. Is this beneficial?

Reply:
It was the only class with this kind of constructor. Again, I think we should either generate such constructors for every class or for none of them. Otherwise we'd have to introduce a special case in the generator just for this one class.

Reply:
At least one more class has it: `ScalaWorkspaceEdit`
I'm mostly agnostic on this, but I wouldn't mind having a "full" convenience constructor for every such class. Would reduce some boilerplate setter calls in tests.

Reply:
But if we include a "full" constructor it will quickly become a problem once we add a new field to an existing struct. Because adding a new field will cause the generator to change the "full" constructor and therefore introduce a breaking change

Reply:
If you are worried about boilerplate in case of code we maintain - in the Kotlin library we'll have data classes. And if someone else wants to get rid of boilerplate in their java code they can contribute a generator for a new library which will use java 17 records ;)

Reply:
> But if we include a "full" constructor it will quickly become a problem once we add a new field to an existing struct

Good argument, so better to remove them fully.